aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -514,13 +571,19 @@ module aptos_framework::account {
     }
 
     /// Revoke the rotation capability offer given to `to_be_revoked_recipient_address` from `account`
-    public entry fun revoke_rotation_capability(account: &signer, to_be_revoked_address: address) acquires Account {
-        assert!(exists_at(to_be_revoked_address), error::not_found(EACCOUNT_DOES_NOT_EXIST));
+    public entry fun revoke_rotation_capability(
+        account: &signer, to_be_revoked_address: address
+    ) acquires Account {
+        assert!(
+            exists_at(to_be_revoked_address), error::not_found(EACCOUNT_DOES_NOT_EXIST)
+        );
```

Comment:
This is another one where different lines may be less confusing

Reply:
Here is single-line mode, according to issue10

Reply:
Could you provide the link to `issue10`?

Reply:
https://github.com/movebit/movefmt/issues/10.